To: vim_dev@googlegroups.com
Subject: Patch 7.3.105
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.105
Problem:    Can't get the value of "b:changedtick" with getbufvar().
Solution:   Make it work. (Christian Brabandt)
Files:      src/eval.c


*** ../vim-7.3.104/src/eval.c	2011-01-17 20:08:03.000000000 +0100
--- src/eval.c	2011-01-22 01:10:42.000000000 +0100
***************
*** 10866,10871 ****
--- 10866,10876 ----
  
  	if (*varname == '&')	/* buffer-local-option */
  	    get_option_tv(&varname, rettv, TRUE);
+ 	else if (STRCMP(varname, "changedtick") == 0)
+ 	{
+ 	    rettv->v_type = VAR_NUMBER;
+ 	    rettv->vval.v_number = curbuf->b_changedtick;
+ 	}
  	else
  	{
  	    if (*varname == NUL)
*** ../vim-7.3.104/src/version.c	2011-01-22 00:58:15.000000000 +0100
--- src/version.c	2011-01-22 01:13:19.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     105,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
133. You communicate with people on other continents more than you
     do with your own neighbors.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
